apps/namadillo/src/hooks/useSdk.tsx

```diff
@@ -47,7 +53,19 @@ export const SdkProvider: FunctionComponent<PropsWithChildren> = ({
 
   useEffect(() => {
     if (nativeToken.data) {
-      getSdkInstance().then((sdk) => setSdk(sdk));
+      getSdkInstance().then((sdk) => {
+        setSdk(sdk);
+        const { masp } = sdk;
+        masp.hasMaspParams().then((hasMaspParams) => {
+          if (!hasMaspParams) {
```

Comment:
I've tried to test it and I think we have to remove this `!` otherwise params will never fetch 🤔

Reply:
yep, sorry, I was doing a test and forgot to remove this. Thanks!

Reply:
Fixed!